src/bindings/js/node/lib/addon.ts

```diff
@@ -36,6 +36,7 @@ interface Core {
     modelBuffer: Uint8Array, weightsBuffer?: Uint8Array): Promise<Model>;
   readModelSync(modelPath: string, weightsPath?: string): Model;
   readModelSync(modelBuffer: Uint8Array, weightsBuffer?: Uint8Array): Model;
+  importModel(modelStream: Buffer, device: string): CompiledModel;
```

Comment:
Are we going to support **properties** parameter?

Reply:
This is an optional parameter. I suggest adding it later with a specific use case in mind.